catroid/src/main/res/values/strings.xml

```diff
@@ -756,6 +756,15 @@
     <string name="upload_tag_dialog_title">Please choose up to three categories</string>
     <string name="upload_tags_maximum_error">Selecting more than 3 tags is not possible.</string>
 
+    <!-- Download ML Module -->
+    <string name="download_ml_module_permission_dialog_title">Download Machine Learning Module?</string>
+    <string name="download_ml_module_permission_dialog_message">Your project requires the Machine Learning Module. Do you want to download it?</string>
+    <string name="download_ml_module_not_available_dialog_title">Machine Learning Feature not available</string>
+    <string name="download_ml_module_not_available_dialog_message">The machine learning features are not available on this device at the moment.</string>
+    <string name="notification_program_name_ml_module">Machine Learning Module</string>
+    <string name="download_ml_module_error_message">Failed to download Machine Learning Module</string>
+    <!--  -->
```

Comment:
```suggestion
     <!-- Download AI module -->
     <string name="download_ml_module_permission_dialog_title">Download Artificial Intelligence (AI) extension module?</string>
    <string name="download_ml_module_permission_dialog_message">Your project requires the Artificial Intelligence (AI) extension module. Do you want to download it?</string>
    <string name="download_ml_module_not_available_dialog_title">Artificial Intelligence (AI) extension feature not available</string>
    <string name="download_ml_module_not_available_dialog_message">The Artificial Intelligence (AI) extension  features are not available on this device at the moment.</string>
    <string name="notification_program_name_ml_module">Artificial Intelligence (AI) extension module</string>
    <string name="download_ml_module_error_message">Failed to download Artificial Intelligence (AI) extension module</string>
```

For our users, this is only known as the AI extension via the settings of the app. Nowhere else is written anything about machine learning, and actually from user perspective these features do not possess any capability to learn anything.